dev-tools/audius-compose

```diff
@@ -294,6 +294,10 @@ def push(ctx, services, prod):
             base_tag = f"{protocol_dir.name}-{service}"
 
             if service == "dashboard" and prod:
+                # tag -dev image (for audius-d)
+                subprocess.run(["docker", "tag", f"{base_tag}:dev", f"audius/{service}:{git_commit}-dev"], check=True)
+                subprocess.run(["docker", "push", f"audius/{service}:{git_commit}-dev"], check=True)
```

Comment:
this line can be removed. it's redundant with the other one below. looks good otherwise!